app/models/user_skill.rb

```diff
@@ -0,0 +1,6 @@
+class UserSkill < ActiveRecord::Base
+  validates_presence_of :skill_id, null:false
+  validates_presence_of :user_id, null:false
```

Comment:
Space missing after colon.<br>Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.